src/posthog-featureflags.ts

```diff
@@ -390,10 +391,12 @@ export class PostHogFeatureFlags {
             data.disable_flags = true
         }
 
+        const eligibleForFlagsV2 = token === DEFAULT_POSTHOG_APP_API_KEY && this.instance.config.__preview_remote_config
```

Comment:
Any chance we could just do

```suggestion
        const eligibleForFlagsV2 = this.instance.config.__preview_flags_v2
```

so that I can run this on localhost easily?

Reply:
oh yeah, and then we can just set the preview to true for our own posthog JS initialization + any other instances of it.

That's a good idea; I might still want to use API tokens down the road (especially when rolling out to specific users), but I'm happy doing something like this instead.  Thanks for the suggestion!

Reply:
@robbie-c went ahead and went with the config-based approach, although I might need to use API tokens down the road too (so I can turn this on for specific customers as needed).  I think my plan will be to have a set of tokens that have this feature toggled on, and then allow folks to specify it in their config as well.